core/conn/odbc/src/odbc/nsksrvr/Makefile

```diff
@@ -130,6 +130,8 @@ LOC_JSIG=$(JAVA_HOME)/jre/lib/i386/server
 endif
 
 
+#gcc 4.8 need explicit enable this flag, otherwise ld must explicitly link with all so using '-l'
```

Comment:
This is NOT a change of gcc4.8, but the CentOS 7. CentOS 7 build gcc with '--no-copy-dt-needed-entries' OFF by default. Turn it on is a simple but not good change. I will come back later when all .cpp/.h changes are merged.